app/tests/integration/nimbus/test_desktop_client_integrations.py

```diff
@@ -124,31 +123,12 @@ def test_check_telemetry_enrollment_unenrollment(
     # Check their was a telemetry event for the enrollment
     control = True
     timeout = time.time() + 60 * 5
-    while control or time.time() < timeout:
-        telemetry = requests.get("http://ping-server:5000/pings").json()
-        for item in reversed(telemetry):
-            if "events" in item["payload"]:
-                if "parent" in item["payload"]["events"]:
-                    for events in item["payload"]["events"]["parent"]:
-                        try:
-                            assert "normandy" in events
-                            assert "enroll" in events
-                            assert "nimbus_experiment" in events
-                            assert experiment_slug in events
-                        except AssertionError:
-                            continue
-                        else:
-                            control = False
-                            break
-            else:
-                continue
-        # If there are no pings we have to wait
-        else:
-            trigger_experiment_loader()
-            time.sleep(15)
-    else:
-        if control is not False:
+    while control is not True:
```

Comment:
why not just write  `while control`?

Reply:
Oh this is wrong. Control should be false. Fixing

Reply:
then `While not control`